documentation/source/user_section/tutorials/gm-wm/gm-wm-metric-computation/csa-gm-wm.rst

```diff
@@ -0,0 +1,21 @@
+Using binary masks to compute CSA for gray and white matter
+###########################################################
+
+The computed GM and WM segmentations can be used to compute the cross sectional area of GM and WM. This is achieved using ``sct_process_segmentation``.
+
+.. code::
+
+   sct_process_segmentation -i t2s_wmseg.nii.gz -o csa_wm.csv -angle-corr 0
+
+.. code::
+
+   sct_process_segmentation -i t2s_gmseg.nii.gz -o csa_gm.csv -angle-corr 0
+
+:Input arguments:
+   - ``-i`` : The input segmentation file.
+   - ``-o`` : The output CSV file.
+   - ``-angle-corr 0``: Normally, angle correction will be applied during ``sct_process_segmentation`` to account for scans where the spinal cord is positioned at an angle with respect to the superior-inferior axis. While this works well when the input is a full spinal cord segmentation, in this case we are instead providing GM/WM segmentations only. The shape of these segmentations can have a negative effect on so the estimation of the cord centerline, which in turn may cause the estimated angle to be incorrect. So, here we specify ``0`` to turn off angle correction.
+   - **Note:** Turning off angle correction is only safe to do if you know that your axial slices were acquired roughly orthogonal to the cord.
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/jn/2857-add-remaining-tutorials/gm-wm-metric-computation/io-sct_process_segmentation.png
```

Comment:
![Screenshot from 2021-08-13 16-00-04](https://user-images.githubusercontent.com/16181459/129412387-613480f7-9bb3-4581-a01d-019bd8e24db6.png)


Yes, good point! I could either...

1. Leave the command as-is. and simply remove the figure
2. Add `-perslice`, then simply change the axis on the graph to show slice #'s
3. Add `-vert` and `-perlevel` and leave the image as-is

I think I like 2 the most? 1 kind of feels a little useless (only a single value!), and 3 would mess up the order of the tutorials (because it would require warping the vertebral levels).

Reply:
agreed with 2!
you could also add a small icon representing a CSV file, to make it clear that this is the output

Reply:
I would also go with 2!

Reply:
[csa_wm_perslice.csv](https://github.com/spinalcordtoolbox/spinalcordtoolbox/files/7033797/csa_wm_perslice.csv)

![io-sct_process_segmentation](https://user-images.githubusercontent.com/16181459/130490116-20d1e5e2-d5f8-40c6-b659-ed5eb498846d.png)

Reply:
Hmm. I'm thinking it might actually be good to have both commands (one with `-perslice` and one without).

The reason for this is that the averaged CSA value is currently used in our `batch_processing.sh` test:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/c5f194410605dd9c5035d704e1656930830a9cdc/testing/batch_processing/test_batch_processing.py#L22

So, if we keep the command with the averaged value, then the SCT Course commands will stay in sync with `batch_processing.sh`, which is relevant for #3508.